libs/ledger-live-common/src/families/cosmos/js-prepareTransaction.ts

```diff
@@ -55,24 +55,28 @@ export const getEstimatedFees = async (
 
   const cosmosAPI = new CosmosAPI(account.currency.id);
   const { protoMsgs } = txToMessages(account, transaction);
-  const { sequence, pubKeyType, pubKey } = await cosmosAPI.getAccount(account.freshAddress);
-  const signature = new Uint8Array(Buffer.from(account.seedIdentifier, "hex"));
+  const { sequence, pubKey, pubKeyType } = await cosmosAPI.getAccount(account.freshAddress);
 
-  const txBytes = buildTransaction({
+  const unsignedTx = buildTransaction({
     protoMsgs,
-    memo: transaction.memo || "",
-    pubKeyType,
-    pubKey,
-    feeAmount: undefined,
     gasLimit: undefined,
-    sequence: sequence ? sequence + "" : "0",
```

Comment:
Can you confirm this works with a fresh account ? (new account, no tx broadcasted yet)

Reply:
Will test and let you know, but that should be catched by the bot right ?

Reply:
I'm asking this because by experience I think sequence is null in that specific case

For the bot idk as I think it does not create new account once it reaches max simultaneous account set by the config

Reply:
Yes, it works as well:

```
curl 'https://cosmoshub4.coin.ledger.com/cosmos/tx/v1beta1/simulate' \
  --data-raw '{"tx_bytes":[10,145,1,10,142,1,10,28,47,99,111,115,109,111,115,46,98,97,110,107,46,118,49,98,101,116,97,49,46,77,115,103,83,101,110,100,18,110,10,45,99,111,115,109,111,115,49,97,117,50,110,115,53,109,114,121,116,112,51,116,109,99,53,55,48,52,115,120,113,107,106,115,115,103,54,113,101,117,108,102,119,114,110,122,122,18,45,99,111,115,109,111,115,49,113,122,106,103,114,113,106,121,56,57,52,106,120,51,108,117,120,102,97,109,114,116,50,110,122,52,114,119,120,55,110,108,56,100,106,121,119,51,26,14,10,5,117,97,116,111,109,18,5,49,54,49,52,49,18,57,10,41,10,33,10,31,47,99,111,115,109,111,115,46,99,114,121,112,116,111,46,115,101,99,112,50,53,54,107,49,46,80,117,98,75,101,121,18,4,10,2,8,127,18,12,10,10,10,5,117,97,116,111,109,18,1,49,26,65,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]}' 
```

(Will not work anymore due to the account not having funds anymore, proof that the simulation works at least 😛 

Transaction: https://www.mintscan.io/cosmos/tx/BBAFC0B63D5D8EFB8E574BCC406F4791A2E48B7847486018FBCDDB5D15AC2012

Reply:
Perfect :ok_hand: